_includes/br/The-shell-aka-REPL.md

```diff
@@ -1,13 +1,13 @@
-|                                        |                              |
-| -------------------------------------- | ---------------------------- |
-| Recall last result                     | `ans`                        |
-| Interrupt execution                    | \[Ctrl\] + \[C\]             |
-| Clear screen                           | \[Ctrl\] + \[L\]             |
-| Run program                            | `include("filename.jl")`     |
-| Get help for `func` is defined         | `?func`                      |
-| See all places where `func` is defined | `apropos("func")`            |
-| Command line mode                      | `;`                          |
-| Package Manager mode                   | \]                           |
-| Help mode                              | ?                            |
-| Exit special mode / Return to REPL     | \[Backspace\] on empty line  |
-| Exit REPL                              | `exit()` or \[Ctrl\] + \[D\] |
+|                                               |                              |
+| --------------------------------------------- | ---------------------------- |
+| Relembrar o último resultado                  | `ans`                        |
+| Interromper a execução                        | \[Ctrl\] + \[C\]             |
+| Limpar tela                                   | \[Ctrl\] + \[L\]             |
+| Executar programa                             | `include("filename.jl")`     |
+| Encontrar ajuda relacionada à `func`          | `?func`                      |
+| Ver todos os locais onde `func` está definida | `apropos("func")`            |
```

Comment:
In both cases, it may be clearer to simply use "função"; simply placing `` `func` `` in the sentence it doesn't work as well in Portuguese as it does in English.
```suggestion
| Encontrar ajuda relacionada a uma função          | `?func`                      |
| Ver todos os locais onde uma função está definida | `apropos("func")`            |
```